_includes/modern-appendix.md

```diff
@@ -438,24 +446,37 @@ Sent as a reply to the [`WHOWAS`](#whowas-message) command, this numeric shows d
 
 ### `RPL_WHOISIDLE (317)`
 
-      "<client> <nick> <secs> [<signon>] :seconds idle, signon time
+      "<client> <nick> <secs> <signon> :seconds idle, signon time
```

Comment:
Missing double quote at end of message. Also applies to RPL_WHOISSPECIAL, RPL_WHOISACTUALLY and RPL_WHOISHOST.